sdk/servicebus/azure-servicebus/samples/async_samples/auto_lock_renew_async.py

```diff
@@ -9,7 +9,10 @@
 Example to show usage of AutoLockRenewer asynchronously:
     1. Automatically renew locks on messages received from non-sessionful entity
     2. Automatically renew locks on the session of sessionful entity
-"""
+
+We do not guarantee that this SDK is thread-safe. We do not recommend reusing the ServiceBusClient,
```

Comment:
maybe you can add the thing about "It is up to the running application..." line here as well?

Reply:
will do